packages/frontend/src/components/common/Table/Table.styles.ts

```diff
@@ -18,6 +18,7 @@ export const TableScrollableWrapper = styled.div`
     overflow: auto;
     min-width: 100%;
     border-radius: 4px;
+    border: 1px solid #dcdcdd;
```

Comment:
border should be applied to the container, not the table - this ensures no hacks. same for pivot